jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -543,6 +560,40 @@ public void setContinuation(Thread thread, Continuation c) {
 		thread.setContinuation(c);
 	}
 
+/*[IF JAVA_SPEC_VERSION >= 20]*/
+	public Object[] scopedValueCache() {
+		return Thread.scopedValueCache();
+	}
+
+	public void setScopedValueCache(Object[] cache) {
+		Thread.setScopedValueCache(cache);
+	}
+
+	public Object scopedValueBindings() {
+		return Thread.scopedValueBindings();
+	}
+
+	public Object findScopedValueBindings() {
+		return Thread.findScopedValueBindings();
+	}
+
+	public void setScopedValueBindings(Object scopeValueBindings) {
+		Thread.setScopedValueBindings(scopeValueBindings);
+	}
+
+	public void ensureMaterializedForStackWalk(Object obj) {
+		Thread.ensureMaterializedForStackWalk(obj);
+	}
+
+	/**
+	 * Returns the initial system in set by System.setIn().
+	 *
+	 * @return the initialSystemIn
+	 */
```

Comment:
As @tajila commented, this isn't really helpful. Please remove it for consistency with other methods here.

Reply:
FYI: @JasonFengJ9

Reply:
Removed

Reply:
Picked up Jason's changes at https://github.com/eclipse-openj9/openj9/pull/16574.